net/nimble/controller/src/ble_ll_sched.c

```diff
@@ -1107,30 +1107,43 @@ ble_ll_sched_execute_item(struct ble_ll_sched_item *sch)
     int rc;
     uint8_t lls;
 
+    lls = ble_ll_state_get();
+    if (lls == BLE_LL_STATE_STANDBY) {
+        goto sched;
+    }
+
+    if (sch->sched_type == BLE_LL_SCHED_TYPE_AUX_SCAN) {
+        if (lls == BLE_LL_STATE_INITIATING || lls == BLE_LL_STATE_SCANNING) {
```

Comment:
priority for init state over aux scan does make sense, but do we need to prioritize regular scan as well? I'd rather think of priority for connection over aux scan.

also some comment would be nice here